sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -39,6 +39,7 @@ DistortionCorrectorComponent::DistortionCorrectorComponent(const rclcpp::NodeOpt
   // Parameter
   time_stamp_field_name_ = declare_parameter("time_stamp_field_name", "time_stamp");
   use_imu_ = declare_parameter("use_imu", true);
+  update_azimuth_and_distance_ = declare_parameter("update_azimuth_and_distance", false);
```

Comment:
As https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/contributing/coding-guidelines/ros-nodes/parameters.md#parameters, I think it is better to remove default param from declare_parameter
```suggestion
  update_azimuth_and_distance_ = declare_parameter<bool>("update_azimuth_and_distance");
```

Reply:
Sure, I can remove the default value. 
I have one question is that should I create the parameter file for this node? 
Currently, the other node in the pointcloud preprocessor doesn't have the parameter file and also uses declare_parameter with the default value.

Reply:
Sure! I will remove the default param! 
Thanks for the advice!